Mg/add component readme (#516)
Esri/jsapi-resources: component-samples/README.md

# ArcGIS Maps SDK for Javascript - Component Samples
 

> This directory contains samples that demonstrate the usage of ***web components*** for the ArcGIS Maps SDK for JavaScript. There are three component packages available:
>  
> - `@arcgis/map-components`
> - `@arcgis/coding-components`
> - `@arcgis/charts-components`

Reviewer: ```suggestion
This directory contains samples that demonstrate how to use web components from the ArcGIS Maps SDK for JavaScript with various frameworks, module bundlers, and build tools. There are three component packages available:
 
- Map components
- Coding components
- Charts components (beta)
```

> These packages can be accessed through the ArcGIS CDN or npm. To install npm, visit [https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm).

Reviewer: ```suggestion
Components from the SDK can be accessed through the ArcGIS CDN or npm. To install npm, visit [https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm).
```

> Each component directory includes samples using CDN and ES module formats, with support for various frameworks, module bundlers, and build tools.

Reviewer: ```suggestion
```
